.envrc

```diff
@@ -12,8 +12,9 @@
 # Environment Node is running in (including build scripts)
 # NOT the same as the environment the application is running in
 # Use "development" for local development with hot reloading, otherwise use "production"
-# Options: development, production
-export NODE_ENV=development
+# Options: development, production, test
```

Comment:
If we're just going to comment this out, why not get rid of it? Isn't that the reason we use source control?

Reply:
Yes true.. I thought the comments were worth keeping though because `NODE_ENV` can be a confusing thing sometimes